### PR TITLE
Docs: add task list in issue templates

### PR DESCRIPTION
**Pull request by hongriTianqi**
By adding task list in issue templates, one can easily check the progress of issues:
截屏2023-09-08 11 11 32

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/questions.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Questions
 description: For users or developers who need help with software installation, usability, or other issues.
-labels: [Questions]
+labels: [Help wanted]
```
Comment (hongriTianqi) [inline .github/ISSUE_TEMPLATE/questions.yml]:
```suggestion
labels: [Questions]

File: .github/ISSUE_TEMPLATE/tests.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: Tests
 description: For developers to report issues related to software testing, such as test failures, missing or incomplete tests, or issues with test automation.
-labels: [Tests]
+labels: [Test]
```
Comment (hongriTianqi) [inline .github/ISSUE_TEMPLATE/tests.yml]:
```suggestion
labels: [Tests]
```

**Review by caic99 — APPROVED** [review]
@hongriTianqi LGTM! But how about using "Issue assignee"? "attacker“ feels aggressive 🤣

**Comment by hongriTianqi** [comment]
> @hongriTianqi LGTM! But how about using "Issue assignee"? "attacker“ feels aggressive 🤣

@caic99 Thanks! I want to use attacker to inspire developers to solve the issue, since the task may not be solved by only one assignee.

**Pull request merged**